### PR TITLE
Stop campaign if budget is reached

### PR DESCRIPTION
**Pull request by sana7044**
Stop campaign if budget is reached

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "projects": {
     "web": "celebrityfanalizer",
-    "default": "celebrityfanalizer"
+    "default": "celebrityfanalyzer-dev"
```
Comment (arnonrdp) [inline .firebaserc:4]: @sana7044, I didn't get this.

Comment (sana7044) [reply .firebaserc:4]: It specifies project ID. This means in which project the functions will be deployed.

Comment (arnonrdp) [reply .firebaserc:4]: I know, I am wondering why we are using the `dev` project now.

Comment (sana7044) [reply .firebaserc:4]: I have access to the CelebrityFanalyzar-Dev project and when I was trying to deploy the cloud functions I would need to specify in which project I needed to deploy those functions so I used dev. In my next commit, I will remove it.

**Review by arnonrdp — APPROVED** [review]

**Pull request merged**